src/server/node/modules/storage/system.js

```diff
@@ -61,12 +61,17 @@ module.exports.setSettings = function(http, username, settings) {
   });
 
   return new Promise(function(resolve, reject) {
-    _fs.writeFile(path, JSON.stringify(settings), function(err, res) {
-      if ( err ) {
-        reject(err);
-      } else {
-        resolve(true);
+    _fs.ensureFile(path, (err) => {
```

Comment:
Could you please change it to `function` instead of an arrow ? I'd like to keep it consistent throughout the codebase.